haiku-apps/xrs/xrs-1.9.0.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="A simple rhythm station for creating music loops"
+DESCRIPTION="XRS is a simple Rhythm Station. You can use it to create music compositions and loops of your preferred elettronic music genres."
+HOMEPAGE="https://github.com/Freaxed/xrs-haiku"
+COPYRIGHT="2006-2022, Andrea Anzani."
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/Freaxed/xrs-haiku/archive/refs/tags/v1.9.0.tar.gz"
+CHECKSUM_SHA256="6ab0b77d32d3b953973ff8457ca98f2ae66e93f4951f76b6615fb9d6476992cb"
+SOURCE_DIR="xrs-haiku-1.9.0"
+
+ARCHITECTURES="x86_64 x86 !x86_gcc2"
```

Comment:
For this to build on 32bit you need to add `SECONDARY_ARCHITECTURES="x86"` and `$secondaryArchSuffix` where needed.
No x86 primary architecture, so you could use either:
 `ARCHITECTURES="x86_64 !x86_gcc2"`
or (how it's done now, not needed as XRS isn't tested on other architectures:
 `ARCHITECTURES="all !x86_gcc2"`

Reply:
Is there a way for me to compile and test it on a x86_gcc2 hybrid installation? I played a bit with haikuporter and the _x86 postfix but no luck so far..

Reply:
If you got a 32bit installation just open a Terminal in the source directory:
1. gcc -v (yields gcc2)
2. setarch x86 (switch to gcc11)
3. gcc -v (yields gcc11)
4. run make from the source

Reply:
Great! thank you!

Reply:
```
 ARCHITECTURES="all !x86_gcc2"
```
is indeed the way to go.